devicons.py

```diff
@@ -138,6 +172,7 @@
     'Desktop'                          : '',
     'Documents'                        : '',
     'Downloads'                        : '',
+    'Dotfiles'                         : '',
```

Comment:
Instead of this being the same icon as dropbox, I think having it be the `cog` icon makes more sense.

Reply:
Ah yes, I agree. I went ahead and changed it.